packages/components/src/internal/components/assay/withAssayModels.tsx

```diff
@@ -238,15 +238,18 @@ export function withAssayModels<Props>(
  * This will have [[InjectedAssayModel]] props injected into it when instantiated.
  * @param defaultProps Provide alternative "defaultProps" for this wrapped component.
  */
+// TODO: this component seems kind of unnecessary, it seems like every consumer should be able to just use the RR6
```

Comment:
With RR6 I fully agree. I'd say just refactor this out. The only two usages are in `@labkey/premium` and they both have already been refatored to use `useParams`.